cmake/onnxruntime_providers_tensorrt.cmake

```diff
@@ -57,16 +56,32 @@
     MESSAGE(STATUS "Can't find NV_TENSORRT_MAJOR macro")
   endif()
 
-  # Check TRT version >= 10.0.1.6
-  if ((NV_TENSORRT_MAJOR_INT GREATER 10) OR
-      (NV_TENSORRT_MAJOR_INT EQUAL 10 AND NV_TENSORRT_MINOR_INT GREATER 0) OR
-      (NV_TENSORRT_MAJOR_INT EQUAL 10 AND NV_TENSORRT_PATCH_INT GREATER 0))
-    set(TRT_GREATER_OR_EQUAL_TRT_10_GA ON)
-  endif()
-
+  # Check TRT version >= requirement
+  function(check_trt_version req_major req_minor req_patch out_var)
```

Comment:
This function is much complex than the original check.
Any reasons to add this function?

Reply:
So far it's not needed. I will clean this